pvlib/pvsystem.py

```diff
@@ -378,6 +378,72 @@ def get_iam(self, aoi, iam_model='physical'):
         return tuple(array.get_iam(aoi, iam_model)
                      for array, aoi in zip(self.arrays, aoi))
 
+    @_unwrap_single_value
+    def get_cell_temperature(self, poa_global, temp_air, wind_speed, model,
```

Comment:
I'm not concerned about the name here because it's just an argument. I'm more concerned about the column name in the dataframe for `run_model` and variants.